docs/dockerfile_to_build_custom_logstash_image.md

```diff
@@ -0,0 +1,72 @@
+# Dockerfile to build a custom Logstash image
```

Comment:
Building Custom Docker Images

Filename can just be `docker.md`.

Reply:
done